impl/src/main/kotlin/com/android/declarative/internal/SettingsDeclarativePlugin.kt

```diff
@@ -201,6 +198,35 @@ class SettingsDeclarativePlugin @Inject constructor(
 
         // only registers the declarative plugin if there is a build.gradle.toml file present in the project dir.
         if (declarativeFile.isPresent) {
+            settingsDeclarations.getArray("plugins")?.forEachTable { table ->
+                // TODO : reconcile and use DependencyParser.
```

Comment:
Nit: Can this plugin array processing code and the one in `else` statement be de-duplicated?